search/includes/classes/commands/class-corecommand.php

```diff
@@ -45,6 +45,10 @@ private function shift_version_after_index( $assoc_args ) {
 
 		foreach ( $indexables as $indexable ) {
 			WP_CLI::line( sprintf( 'Updating active version for "%s"', $indexable->slug ) );
+			if ( ! $skip_confirm ) {
+				WP_CLI::confirm( 'Please confirm' );
```

Comment:
Just a minor suggestion to clarify what the user is confirming
```suggestion
				WP_CLI::confirm( 'Please confirm you'd like to update active version' );
```